### PR TITLE
Add READMEs to pkg packages

### LINKED ISSUE #1298: Add README documentation for packages in the `pkg` dir
**ENHANCEMENT**
1. Describe the enhancement you are requesting.
Packages in the `pkg` directory are missing README file doc. Please add it to each of them.

2. Indicate the importance of this enhancement to you (must-have, should-have, nice-to-have).
must-have

### PR DESCRIPTION
**Pull request by darkodraskovic**
Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

### What does this do?

Add `README.md` pages to the `pkg` packages.

### Which issue(s) does this PR fix/relate to?
Resolves #1298

### CODE REVIEW

File: pkg/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Standalone packages
+
+The `pkg` directory (the current directory) contains a set of standalone packages that can be imported and used by any applications. The packages are specifically meant for the development of the Mainflux based back-end applications and implement common tasks needed by the programmatic operation of Mainflux platform.
```
Comment (drasko) [inline pkg/README.md]: `by any application`

Comment (drasko) [reply pkg/README.md]: Also, as a reference, maybe you can take a look here at the explanation of the role of this standard directory: https://github.com/golang-standards/project-layout/tree/master/pkg

File: pkg/auth/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Auth client
+
+Auth client package is used to identify a thing and authorize a thing's access to a channel.
+
+To identify a thing, you need a valid **thing key**. You retrieve thing's identity in the form **thing id**, that can be used for thing's and thing's connections' CRUD operations.
```
Comment (drasko) [inline pkg/auth/README.md]: `thing connection's` probably?

File: pkg/sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Mainflux SDK kits
+
+This directory contains drivers for Mainflux HTTP API. Drivers do system administration (provisioning) and messaging. They are written in different languages in order to enable the faster application development in the respective language.
+
+Currently, there is only a fully developed SDK in Go. SDK's in Python and JavaScript are on the road map and are currently in development.
```
Comment (drasko) [inline pkg/sdk/README.md]: I would remove this sentence, as once other SDK is merged, we'll forget to remove it.

**Review by drasko — CHANGES_REQUESTED** [review]

File: pkg/auth/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Auth client
+
+Auth client package is used to identify a thing and authorize a thing's access to a channel.
+
+To identify a thing, you need a valid **thing key**. You retrieve thing's identity in the form of a **thing id**. The latter is used in CRUD operations on things and their connections.
```
Comment (manuio) [inline pkg/auth/README.md]: `thing ID`

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]

**Pull request merged**